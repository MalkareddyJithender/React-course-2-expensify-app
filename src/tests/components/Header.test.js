import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';


test('should render header component',() =>
{
   //const renderer = new ReactShallowRenderer();
   //console.log(renderer.render(<Header />));
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    const wrapper = shallow(<Header />);
    //expect(wrapper.find('h1').text()).toBe('Expensify');
    expect(wrapper).toMatchSnapshot();
   
});