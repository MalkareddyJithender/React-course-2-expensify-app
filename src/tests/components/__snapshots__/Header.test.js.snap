// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    DashboardExpensepage
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    AddExpensePage
  </[object Object]>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
